src/api/.rubocop_todo.yml

```diff
@@ -293,6 +293,7 @@ Metrics/CyclomaticComplexity:
     - 'app/models/project_status/calculator.rb'
     - 'app/models/relationship.rb'
     - 'app/models/repository.rb'
+    - 'app/models/search_finder.rb'
```

Comment:
It's better now, but there's still work to reduce the complexity. I'm fine with that :+1: